run_conode.sh

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env bash
+set -e
+
+VERSION=1.0-pre1
+MAILCMD=mail
+MAILADDR=linus.gasser@epfl.ch
```

Comment:
Do you have any reference as how to use run_conode ? in the README ? 
If so, do you specifiy that it sends a log to this address automatically if mail is installed ?